chord_metadata_service/discovery/schemas.py

```diff
@@ -0,0 +1,110 @@
+from pathlib import Path
+from chord_metadata_service.restapi.schema_utils import (
+    get_schema_app_id, sub_schema_uri, array_of, base_type, SchemaTypes, enum_of
+)
+
+base_uri = get_schema_app_id(Path(__file__).parent.name)
+
+DISCOVERY_FIELD_SCHEMA = {
+    "$id": sub_schema_uri(base_uri, "discovery_field"),
```

Comment:
it would be good for this ID to be a true URL of the schema; but maybe this isn't so straightforward to do right now?

Reply:
Schemas for discovery, phenopacket and experiment now use a true URL as the $id, with the accompanying views to resolve schemas/subschemas